client/lib/api/symptom_logic.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter/material.dart';
```

Comment:
This should go in to the `client/flutter/lib/api/display_conditions.dart` file. It fits it with the cohort experimentation logic.

@advayDev1 moved the logic to `display_conditions.dart` in 1289:
https://github.com/WorldHealthOrganization/app/pull/1289/files#diff-773a1ae5f46e15d88eaf0fa31082951d904234896ec9456c8b5cef9ebb292056)

The removed it with 1476:
https://github.com/WorldHealthOrganization/app/pull/1476/files#diff-773a1ae5f46e15d88eaf0fa31082951d904234896ec9456c8b5cef9ebb292056